roofnet/README.md

```diff
@@ -0,0 +1,63 @@
+# Summary
+Roofnet is a wireless mesh architecture that provides high performance Internet access while demanding little deployment planning or operational management.
+
+## Goals
+1. Volunteer users host nodes at home. Thus, no central control over topology, and no central planning.
+2. Omnidirectional antennas that are low quality and probably have high interference.
+3. Multi-hop routing.
+4. High TCP throughout 
+
+## Node Addresses
+Wireless interface IP addresses are auto configured, using a private prefix (e.g. 10) for the high byte, and the lower 24 bits of ethernet addresses for the lower three bytes.
```

Comment:
All this prose could be condensed:
* Wireless interface IP based on hardware ethernet address for low bytes -> no config
* Every box has a NAT for its users -> no address allocation necessary